.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Report a bug / crash
+description: A bug is when the program is not behaving as you'd expect while a crash is defined as the case where the program exits unexpectedly (Segmentation fault, etc.).
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: jank health check
+      description: |
+        Output from the `jank check-health` command.
+
+        While not required in all cases, including this in bug reports can help save everyone time 🙂.
```

Comment:
Is there any way we can ensure this gets put into a code fence? Interpreting the output as markdown isn't going to work well, I think.